locales/en-US.json

```diff
@@ -313,10 +326,10 @@
           "Take the Ferry."
         ],
         "1": [
-          "Take the Millersburg Ferry."
```

Comment:
changing the example phrases makes them incorrect, confusing, and less useful

Reply:
This is due to using the same `<STREET_NAMES>` token in all examples. In this case it looks like we expect `STREET_NAMES=Millersburg Ferry` for this phrase `1` to be correct. 

Is `Millersburg Ferry` a valid street name for the enter_ferry phrase? If so, I can update to use different tokens in the example phrases, and keep them unchanged from before this PR.

In general, the reason I wrote a script to automatically generate these examples is that translating this many example phrases is error-prone and a lot of additional work for translators. By having a script author the examples we can have translators simply review the example phrases for correctness.

Reply:
Discussed with @dgearhart and we think two changes are needed here.
1. Example tokens should be specific to the phrases, since they were chosen to have specific meaning depending on the phrase. 
2. Example phrases should be generated by odin, not by the python build script. Reason is odin has special rules for different phrases, and in the future will build in grammar rules like [declension](https://en.wikipedia.org/wiki/Declension). These examples should still not be translated by translators, but should be reviewed by translators after being generated by odin.